libpermian/reportsenders/base.py

```diff
@@ -116,16 +134,24 @@ def processResult(self, crcUpdate):
             if self.reporting.data is None or self.reporting.data.get('submit_issues', True):
                 for issue in self.issuesFor([crcUpdate]):
                     issue.submit()
-            self.processFinalResult(crcUpdate)
-            # Catch end of test case
-            if all([crc.result.final for crc in self.caseRunConfigurations if crcUpdate.testcase == crc.testcase]):
-                self.processCaseRunFinished(crcUpdate.testcase.name)
-            # Catch end of testun
-            if all([crc.result.final for crc in self.caseRunConfigurations]):
-                self.processTestRunFinished()
-                return True
+
+            if not self.throttleInterval:
```

Comment:
I'm not a big fan of those `if not self.throttleInterval` and `if self.throttleInterval` as the only common part is updating the issues. But it could stay this way for a moment, but this sounds very much to me like something worth refactoring.